lib/highlight.js

```diff
@@ -5,14 +5,21 @@ var Entities = require('html-entities').XmlEntities;
 var entities = new Entities();
 var alias = require('../highlight_alias.json');
 
-hljs.configure({
-  classPrefix: ''
-});
+// hljs.configure({
+//   classPrefix: ''
+// });
```

Comment:
Could you please remove the comment out codes?

Reply:
Done :)